rskj-core/src/jmh/resources/conf/testnet.conf

```diff
@@ -0,0 +1,6 @@
+address=0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826
```

Comment:
Please change this property value to `TBD` as most probably this account won't exist on Testnet. Same for Mainet. We will later think on how to solve it for these cases as they are real networks.